src/components/View/index.js

```diff
@@ -39,6 +39,7 @@ class View extends Component {
     const {
       hitSlop,
       style,
+      domRef,
```

Comment:
I had wrtiten a comment around this, onLayout does not fire and measure doesn't work on one of the views that I created. Needed dom node for clientWidth, let me know the alternative